src/layman/upgrade/upgrade_v1_18.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from db import util as db_util
+from layman import settings
+from layman.layer import LAYER_TYPE
+from layman.map import MAP_TYPE
+
+logger = logging.getLogger(__name__)
+DB_SCHEMA = settings.LAYMAN_PRIME_SCHEMA
+
+
+def adjust_db_for_image_mosaic():
+    logger.info(f'    Alter DB prime schema for image_mosaic')
+
+    statement = f'''
+    ALTER TABLE {DB_SCHEMA}.publications ADD COLUMN IF NOT EXISTS
+    image_mosaic boolean;'''
+
+    db_util.run_statement(statement)
+
+
+def update_db_for_image_mosaic():
```

Comment:
Maybe whole update_db_for_image_mosaic is not necessary if we use default value False in adjust_db_for_image_mosaic.